pygeoapi/api.py

```diff
@@ -569,10 +569,13 @@ def get_collection_items(self, headers, args, dataset, pathinfo=None):
             LOGGER.error(exception)
             return headers_, 500, json.dumps(exception)
 
-        LOGGER.debug('processing property parameters')
-        for k, v in args.items():
-            if k not in reserved_fieldnames and k in p.fields.keys():
-                properties.append((k, v))
+        if self.config['datasets'][dataset]['provider']\
+                .get('allow_property_filtering', False):
+            LOGGER.debug('processing property parameters if enabled')
+            for k, v in args.items():
+                if k not in reserved_fieldnames and k in p.fields.keys():
+                    LOGGER.debug('Add property filter {0}={1}'.format(k, v))
```

Comment:
For consistency, would prefer:     `LOGGER.debug('Add property filter {}={}'.format(k, v))`